shared/openId4VCI/Utils.ts

```diff
@@ -161,11 +161,13 @@ export const getCredentialIssuersWellKnownConfig = async (
   defaultFields: string[],
   credentialConfigurationId: string,
   format: string,
+  vcMetadata: any,
```

Comment:
do we need entire vcMetadata here?
if it's needed. can't we specify type as VCMetadata instead of any?